addons/chat/functions/fn_cmndRP.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Name = TMF_chat_fnc_cmndRP
+ * Author = Freddo
+ *
+ * Syntaxes:
+ * #rp - Resets player position.
+ *
+ * Return Value:
+ * Void
+ *
+ * Description:
+ * See Syntaxes
+ */
+
+#include "\x\tmf\addons\chat\script_component.hpp"
+
+IS_CMND_AVAILABLE(GVAR(rpUsage),"#rp");
+
+private _safePos = (getPosATL player) findEmptyPosition [0, 25, (typeOf player)];
```

Comment:
suggest using `[] call CBA_fnc_currentUnit` to get player as would handle case of zeus usage.